src/lib/mac/blake2mac/info.txt

```diff
@@ -0,0 +1,7 @@
+<defines>
+BLAKE2BMAC -> 20202311
```

Comment:
Date format is YYYYMMDD not YYYYDDMM

(I should probably add a check for this in configure)

Reply:
Yeah I tried to guess what it was haha